### PR TITLE
Small optimizations

### PR DESCRIPTION
**Pull request by Bara**
Mergeable. When we keep this open, it's a never ending story.

### CODE REVIEW

**Comment by Bara** [comment]
A lot of similar things will be pushed into the threaded SQL branch, to avoid merge conflicts.

File: addons/sourcemod/scripting/surftimer/timer.sp [hunk 1]
```diff
@@ -313,6 +324,7 @@ public Action StyleReplayTimer(Handle timer, Handle pack)
 	ResetPack(pack);
 	int client = GetClientOfUserId(ReadPackCell(pack));
 	int style = ReadPackCell(pack);
+	delete pack;
```
Comment (PwnK) [inline addons/sourcemod/scripting/surftimer/timer.sp:327]: unnecessary

File: addons/sourcemod/scripting/surftimer/timer.sp [hunk 2]
```diff
@@ -298,6 +308,7 @@ public Action BonusReplayTimer(Handle timer, Handle pack)
 	ResetPack(pack);
 	int client = GetClientOfUserId(ReadPackCell(pack));
 	int zGrp = ReadPackCell(pack);
+	delete pack;
```
Comment (PwnK) [inline addons/sourcemod/scripting/surftimer/timer.sp:311]: unnecessary

File: addons/sourcemod/scripting/surftimer/hooks.sp [hunk 3]
```diff
@@ -1454,6 +1459,8 @@ public Action DelayedVelocityCap(Handle timer, Handle pack)
 	ResetPack(pack);
 	int client = ReadPackCell(pack);
 	float speedCap = ReadPackFloat(pack);
+	delete pack;
```
Comment (PwnK) [inline addons/sourcemod/scripting/surftimer/hooks.sp:1462]: unnecessary

File: addons/sourcemod/scripting/surftimer/timer.sp [hunk 4]
```diff
@@ -639,6 +673,7 @@ public Action SpecBot(Handle timer, Handle pack)
 	ResetPack(pack);
 	int client = GetClientOfUserId(ReadPackCell(pack));
 	int bot = ReadPackCell(pack);
+	delete pack;
```
Comment (PwnK) [inline addons/sourcemod/scripting/surftimer/timer.sp:676]: unnecessary

**Comment by Bara** [comment]
@PwnK Ups, you're right.

**Comment by Bara** [comment]
Latest version: 
[SurfTimer.zip](https://github.com/olokos/Surftimer-olokos/files/4605927/SurfTimer.zip)

**Pull request merged**